server/controllers/article.controller.js

```diff
@@ -20,25 +20,123 @@ class ArticleController {
     */
     if (req.file) req.body.image = req.file.secure_url;
     const {
-      title, body, tags, description, image, published
+      title, body, tagsList, description, published
     } = req.body;
     const { id } = req.user;
     const successMessage = 'Article successfully created';
     const errorMessage = 'Article was not created successfully';
+    const tags = [...new Set(tagsList)];
     const article = await Article.create({
       title,
       body,
-      tags,
+      tagsList,
       description,
-      image,
+      image: req.body.image,
```

Comment:
Why didn't you destructure `image`?

Reply:
I destructed initially, but the name image was conflicting with the image I destructured for the response once the user has created the article, just to avoid the conflict

Reply:
Research on how to destructure and rename at the same time

Reply:
Ok would do that now